src/test/shared/utilities/textUtilities.test.ts

```diff
@@ -92,18 +92,28 @@ describe('getStringHash', async function () {
 })
 
 describe('getRelativeDate', function () {
-    const now = new Date(2020, 2, 2, 2, 2, 2)
+    const now = new Date(2020, 4, 4, 4, 4, 4) // adjusts for clock skew modifier in `getRelativeDate` fn.
```

Comment:
is this testing clock skew, or working around it?

Reply:
It works around it for the tests just to make them clearer. The intent with the tests is to drive home some absolutes, while in reality it'll be a lot less likely that 5 seconds will make a huge difference for a user.